EasyClangComplete.sublime-settings

```diff
@@ -99,5 +99,8 @@
 
   // Controls if libclang will cache the results.
   // This works faster, but in rare cases can generate wrong completions.
-  "use_libclang_caching": true
+  "use_libclang_caching": true,
+
+  // Enable showing phantoms instead of popups for errors
+  "show_phantoms_for_errors": false
```

Comment:
Maybe we should consider changing the setting for errors to smth like `"error_style"` that would have 3 options:
- `None`
- `Popup`
- `Phantom`

But we can always do this later. No need to change it in this PR.